### ISSUE TITLE
5 days after ROA creation RPKI showing as invalid on route-views.linx.routeviews.org

### ISSUE THREAD

**Issue by mivsvit** [routeviews/public-issues#133]
### Observed Behavior

ROAs created 2024-02-28 15:24 CET+01:00

5 days after adding ROA, prefix still shows  `rpki validation-state: invalid` in output of `sh bgp ipv6 <prefix>` on  route-views.linx.routeviews.org.

Example prefixes:
https://irrexplorer.nlnog.net/prefix/193.36.4.0/22
https://irrexplorer.nlnog.net/prefix/2a0b:6dc0::/29

### Expected Behavior

After waiting a reasonable time (about 24 hours maybe?), prefix should show `rpki validation-state: valid` in output of `sh bgp ipv6 <prefix>`, like on other collectors such as route-views.routeviews.org

### Relevant console output or logs

```shell
route-views.linx.routeviews.org> sh bgp ipv6 2a0b:6dc0::/29
BGP routing table entry for 2a0b:6dc0::/29, version 91154081
Paths: (30 available, best #5, table default)
  Not advertised to any peer
  59605 211393
    2001:7f8:4::3:39c1:1 from 2001:7f8:4::e8d5:1 (151.248.97.2)
    (fe80::2cc:34ff:fee7:ec7c) (used)
      Origin IGP, valid, external, rpki validation-state: invalid
      Last update: Mon Mar  4 12:33:34 2024
route-views.linx.routeviews.org> sh ip bgp 193.36.4.0/22
BGP routing table entry for 193.36.4.0/22, version 70813873
Paths: (38 available, best #6, table default)
  Not advertised to any peer
  13237 211393
    195.66.231.28 from 195.66.224.99 (82.197.136.2)
      Origin IGP, valid, external, rpki validation-state: invalid
      Community: 13237:40044 13237:46442
      Last update: Mon Mar  4 10:43:43 2024

route-views>sh bgp ipv6 2a0b:6dc0::/29
BGP routing table entry for 2A0B:6DC0::/29, version 39011891
Paths: (13 available, best #11, table default)
  Not advertised to any peer
  Refresh Epoch 1
  1351 174 1299 211393
    2620:104:E000:1000::3 from 2620:104:E000:1000::3 (132.198.255.253)
      Origin IGP, localpref 100, valid, external
      path 7F162DC81CB8 RPKI State valid
      rx pathid: 0, tx pathid: 0
  Refresh Epoch 1

  route-views>sh ip bgp 193.36.4.0/22
BGP routing table entry for 193.36.4.0/22, version 71414115
Paths: (20 available, best #19, table default)
  Not advertised to any peer
  Refresh Epoch 1
  3257 1299 211393
    89.149.178.10 from 89.149.178.10 (213.200.83.26)
      Origin IGP, metric 10, localpref 100, valid, external
      Community: 3257:8794 3257:30052 3257:50001 3257:54900 3257:54901
      path 7F16B421FA08 RPKI State valid
      rx pathid: 0, tx pathid: 0
  Refresh Epoch 1
```

**Comment by hanskuhn** [comment]
Thanks for the report. We'll take a look and let you know.

**Comment by hanskuhn** [comment]
I haven't reached a root cause, but searching says there might be a bug in FRR that results in RPKI validation not starting during boot, but manually starting it works just fine, i.e. `rpki stop` then `rpki start`.

Thanks for reporting -- things look OK now.

```
route-views.linx.routeviews.org# show rpki cache-connection
Connected to group 1
rpki tcp cache 128.223.157.xx 3323 pref 1 (connected)
rpki tcp cache 128.223.157.yy 3323 pref 2
```

```
route-views.linx.routeviews.org# sh bgp ipv6 2a0b:6dc0::/29
BGP routing table entry for 2a0b:6dc0::/29, version 91154081
Paths: (30 available, best #5, table default)
  Not advertised to any peer
  1031 211393
    2001:7f8:4::3:39c1:1 from 2001:7f8:4::407:1 (10.10.78.241)
      Origin IGP, valid, external, rpki validation-state: valid
      Community: 1031:804 8714:65010 8714:65012
      Large Community: 8714:1000:1 8714:1001:2
      Last update: Mon Mar  4 17:29:42 2024
```
```
route-views.linx.routeviews.org# sh ip bgp 193.36.4.0/22
BGP routing table entry for 193.36.4.0/22, version 70813873
Paths: (37 available, best #5, table default)
  Not advertised to any peer
  13237 211393
    195.66.231.28 from 195.66.224.99 (82.197.136.2)
      Origin IGP, valid, external, rpki validation-state: valid
      Community: 13237:40044 13237:46442
      Last update: Mon Mar  4 10:43:43 2024
```
